packages/local-serverless-server/README.md

```diff
@@ -0,0 +1,71 @@
+# @hubspot/local-serverless-server
+
+A server to test CMS serverless functions locally.
+
+## Getting started
+
+For more information on using these tools, see [Local Development Tooling: Getting Started](https://designers.hubspot.com/tutorials/getting-started-with-local-development)
+
+### Installation
+
+#### Using `yarn`
+
+```bash
+yarn add @hubspot/cms-cli --dev
+```
+
+#### Using `npm`
+
+```bash
+npm install @hubspot/cms-cli
+```
+
+### Usage
+To start the server, the `start` method can be imported from the `@hubspot/local-serverless-server` package and run with settings like so...
+
+```bash
+const { start } = require('@hubspot/local-serverless-server');
+
+start({
+  accountId: <portalId/accountId>,                                                  // default: 123456
+  contact: <booleanValueToSpecifyIfContactDataShouldBePassedToServerlessFunction>,  // default: true
+  path: <pathToLocalDotFunctionsFolder>,                                            // required
+  port: <customPortToRunServerOn>                                                   // default: 5432
+});
+```
+
+### Mocked Data
+Some of the data that is passed to the serverless function context is mocked. Specifically the `contact` and `limits` properties. It is possible
+to modify the mocked data values by setting values within the `environment` property of the `serverless.json` file.
+
+The environment variables to use to modify the data are:
+
+```
+HUBSPOT_LIMITS_TIME_REMAINING       // default: 600000
+HUBSPOT_LIMITS_EXECUTIONS_REMAINING // default: 60
+HUBSPOT_CONTACT_VID                 // default: 123
+HUBSPOT_CONTACT_IS_LOGGED_IN        // default: false
+HUBSPOT_CONTACT_LIST_MEMBERSHIPS    // default: []
+```
+
+Usage example:
+
+```
+{
+  "runtime": "nodejs12.x",
+  "version": "1.0",
+  "environment": {},
+  "secrets": [],
+  "endpoints": {
+    "myCustomEndpoint": {
+      "method": ["GET"],
+      "file": "myCustomEndpoint.js",
+      "environment": {
+        "HUBSPOT_CONTACT_VID": 456,
+        "HUBSPOT_CONTACT_IS_LOGGED_IN": true,
+        "HUBSPOT_CONTACT_LIST_MEMBERSHIPS": ["some", "memberships"]
+      }
```

Comment:
Can you explain this a bit more? I thought we were faking the way this works when executing in the HubSpot serverless infra?

Reply:
The way production works is it grabs the cookie value for `hubspotutk` which is not an actual contact id. It's some sort of hashed value that the BE understands. AFAIK there is not an API endpoint to get the contact data using this value, and I'm not sure we want there to be an endpoint for this.

So instead, in order to facilitate testing around the `contact` prop that gets passed into the function, this allows for customization of the data that is passed in. By setting values for these environment variables, that mocked value will be passed into the function for testing.